README.md

```diff
@@ -309,6 +309,7 @@ Options:
   --bootscript=""       Assign a bootscript
   -e, --env=""          Provide metadata tags passed to initrd (i.e., boot=resue INITRD_DEBUG=1)
   -h, --help=false      Print usage
+  --local-ssh-key=false Creates an ssh key that you can use to connect on the image
```

Comment:
the main goal of this function is not to create a local ssh key, but to use a local ssh key, and create a new one if no one was found, try to reformulate the description to keep the priority of the features